### PR TITLE
comment fix

### PR DESCRIPTION
**Pull request by Kingwl**

### CODE REVIEW

**Comment by bao-qian** [comment]
Thanks.

File: Wox.Infrastructure/Hotkey/GlobalHotkey.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ public SpecialKeyState CheckModifiers()
             }
             if ((InterceptKeys.GetKeyState(VK_WIN) & 0x8000) != 0)
             {
-                //ALT is pressed
+                //WIN is pressed
```
Comment (vizee) [inline Wox.Infrastructure/Hotkey/GlobalHotkey.cs:66]: it is helpful to me

Comment (zapline) [inline Wox.Infrastructure/Hotkey/GlobalHotkey.cs:66]: it is helpful to me too

Comment (AngryPowman) [inline Wox.Infrastructure/Hotkey/GlobalHotkey.cs:66]: it is helpful to me too too

Comment (ly0) [reply Wox.Infrastructure/Hotkey/GlobalHotkey.cs:66]: it is helpful to me three.

Comment (bao-qian) [reply Wox.Infrastructure/Hotkey/GlobalHotkey.cs:66]: hi, why you guys thinking this is helpful? are you writing code using `InterceptKeys`?

**Pull request merged**